style.css

```
@@ -3623,6 +3623,14 @@ input:checked + .toggle:before {
 	text-overflow: ellipsis;
 }
 
+.shortcutSettingsEntry input:hover {
+    animation: clipText 0s 150ms forwards;
+}
+
+@keyframes clipText {
```

Stylelint (17.9.0)

[error] Expected keyframe name "clipText" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)